src/v2/functions/stat-calculator-service.ts

```
@@ -0,0 +1,151 @@
+    static calculateHealth(unitId: string, rarity: Rarity, rarityStars: RarityStars, rank: Rank): number {
+        const units: IUnitData[] = StaticDataService.unitsData.filter(u => u.id === unitId);
+        return units.length != 1 ? -1 : this.calculateStat(units[0].health, unitId, rarityStars, rank);
+    }
+
+    /**
+     * @returns the calculated damage for the given unit at the given rarity
+     *          and rank. -1 if the unit can't be found.
+     */
+    static calculateDamage(unitId: string, rarity: Rarity, rarityStars: RarityStars, rank: Rank): number {
+        const units: IUnitData[] = StaticDataService.unitsData.filter(u => u.id === unitId);
+        return units.length != 1 ? -1 : this.calculateStat(units[0].damage, unitId, rarityStars, rank);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)